### PR TITLE
Avoid using deprecated packages

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Replace use cases of deprecated packages
- Remove deprecated packages

Related issue: https://github.com/ainblockchain/ain-blockchain/issues/321

### CODE REVIEW

File: afan_client/index.js [hunk 1]
```diff
@@ -30,11 +29,8 @@ class AfanClient {
       const investorsResponse = await requestManager.getInvestors(to);
       investors = investorsResponse.result;
     } catch (err) {
-      if (err instanceof errors.StatusCodeError) {
-        investors = null;
-      } else {
-        throw err
-      }
+      console.log(err);
+      throw err
```
Comment (minsulee2) [inline afan_client/index.js:33]: What about adding ; up?

File: afan_client/index.js [hunk 2]
```diff
@@ -66,9 +62,8 @@ class AfanClient {
         return {code: -4, message: 'Already proposed'};
       }
     } catch (err) {
-      if (!(err instanceof errors.StatusCodeError)) {
-        throw err
-      }
+      console.log(err);
+      throw err
```
Comment (minsulee2) [inline afan_client/index.js:66]: ditto

**Review by minsulee2 — APPROVED** [review]
LGTM with minor comments!

File: afan_client/index.js (hunk 1 above)
Comment (platfowner) [reply afan_client/index.js]: Done. Thanks.

File: afan_client/index.js (hunk 2 above)
Comment (platfowner) [reply afan_client/index.js]: ditto

**Pull request merged**